packages/datatrak-web/src/views/WelcomeScreens/CarouselDots.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2024 Beyond Essential Systems Pty Ltd
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+  gap: 0.75rem;
+  padding-block-end: 1rem;
+`;
+
+const Dot = styled.button<{ $active: boolean }>`
+  width: 0.5rem;
+  height: 0.5rem;
+  border-radius: 50%;
```

Comment:
Doesn’t matter so much here because the width and height are hard-coded to be square, but using % values _usually_ yields unwanted oval-ish results. This is a little more broadly applicable:

```suggestion
  border-radius: calc(infinity * 1px);
```